### ISSUE TITLE
Should not flag @type and @id as missing from context

### ISSUE THREAD

**Issue by gjgd** [gjgd/jsonld-checker#14]
```json
{
  "@context": [
    "https://schema.org",
    "https://identity.foundation/EcdsaSecp256k1RecoverySignature2020/lds-ecdsa-secp256k1-recovery2020-0.0.jsonld",
    "https://identity.foundation/SchnorrSecp256k1Signature2019/contexts/schnorr-v1.json"
  ],
  "@type": "SpecialAnnouncement",
  "name": "Stanford announce COVID-19 testing facility",
  "text": "Stanford Health Care’s same-day primary care program is offering drive-through testing, by appointment, for SARS-CoV-2, the coronavirus that causes COVID-19.",
  "datePosted": "2020-03-16",
  "url": "http://med.stanford.edu/news/all-news/2020/03/stanford-offers-drive-through-coronavirus-test.html",
  "category": "https://www.wikidata.org/wiki/Q81068910",
  "announcementLocation": {
    "@type": "CovidTestingFacility",
    "name": "Stanford Health Care",
    "url": "https://stanfordhealthcare.org/"
  },
  "proof": {
    "type": "EcdsaSecp256k1Signature2019",
    "created": "2020-06-15T14:40:45Z",
    "verificationMethod": "did:example:123#EcdsaSecp256k1VerificationKey2019",
    "proofPurpose": "assertionMethod",
    "jws": "eyJhbGciOiJFUzI1NksiLCJiNjQiOmZhbHNlLCJjcml0IjpbImI2NCJdfQ..9myx7PPj5vvs0KoGs5g-j1SptkS5wVrH4lGwuxisQGAmKxKpmIQ0T3OQaquyV5FtMn2aXA5AqWI7SHfV1bWSLA"
  }
}
```

**Comment by gjgd** [comment]
This object should be valid, but is flagged as invalid because of @type
